frontend/simple/views/SignUp.vue

```diff
@@ -69,7 +69,7 @@ export default {
   mixins: [loginLogout],
   methods: {
     submit: function () {
-      console.log(this.errors)
+     console.log(this.errors)
```

Comment:
I'm not sure why `standard` isn't catching this, but there should be an extra space to align it.